### PR TITLE
Fix 'balena app' (rm, restart, info) with numeric app IDs

### PR DESCRIPTION
**Pull request by pdcastro**
Resolves: #1815
Change-type: patch

### CODE REVIEW

File: lib/utils/validation.ts [hunk 1]
```diff
@@ -88,3 +88,14 @@ export function parseAsInteger(input: string, paramName?: string) {
...
 	return Number(input);
 }
+
+export function tryAsInteger(
+	input: string,
+	paramName?: string,
+): number | string {
+	try {
+		return parseAsInteger(input, paramName);
+	} catch (err) {
```
Comment (Page-) [inline lib/utils/validation.ts:98]:
```suggestion
	} catch {
```

Comment (pdcastro) [reply lib/utils/validation.ts]: Done - thanks for the review! 👍

**Comment by pdcastro** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
@balena-ci retest

**Pull request merged**